packages/pangea-sdk/pangea/deep_verify.py

```diff
@@ -102,10 +103,8 @@ def verify_hash(data: dict, data_hash: str) -> bool:
     """Verify the hash of an event"""
     succeeded = False
     try:
-        data_canon = audit_util.canonicalize_json(data)
-        computed_hash_dec = audit_util.hash_bytes(data_canon)
-        data_hash_dec = audit_util.decode_hash(data_hash)
-        if computed_hash_dec != data_hash_dec:
+        if not audit_util.verify_envelope_hash(EventEnvelope(**data), data_hash):
+            print("Hash failed: ", data)
```

Comment:
Can this be changed to "Hash verification failed: "? That would make the error clearer.